pkgs/applications/misc/web-media-controller/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, glib, pcre, json-glib, gobject-introspection }:
+
+stdenv.mkDerivation rec {
+  pname = "wmc-mpris";
+  name = "${pname}-unstable";
+  version = "2019-04-12";
+
+  src = fetchFromGitHub {
+    owner = "f1u77y";
+    repo = pname;
+    rev = "fcfb1d8f708b8e69bc2ab8b9bc0e2605d9ace585";
+    sha256 = "0plknl1r3z58sl68s60z4cxlwk7z5ld09lza74x5m8knhr94vwy9";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig glib pcre json-glib gobject-introspection ];
```

Comment:
I think `glib pcre json-glib` should be in `buildInputs`.

Reply:
Correct. Thanks for catching that, pushed a fix.